### ISSUE TITLE
Window Previews?

### ISSUE THREAD

**Issue by rik-shaw** [nsz32/nmt-plugin#4]
I am guessing having "hover window previews" is far from "minimalist" but I am curious if you would consider such a feature?

**Comment by nsz32** [comment]
Exact, I don't plan to implement windows preview as it is out of scope for now and would involve heavy libraries additions. But, the plugin allows to quick switching between windows : scroll over a group or, keep click maintained over around the drop-down lists of windows.

Thanks for your interest,

**Comment by Hayao0819** [comment]
I also want you to add this function.
This feature brings you closer to the behavior of the Windows taskbar and makes it even better.

**Comment by nsz32** [comment]
I will probably not implement it and doesn't have the time for. At start I just wanted to create a taskbar for my needs, and I must say it start to meet my expectations very well. But this is Free software, so maybe someone else would try to implement it ?
